tests/integration/_experimental/object_detection/test_evaluator_multiclass_test_case_plots.py

```diff
@@ -0,0 +1,477 @@
+                        ),
+                        Curve(
+                            x=[
+                                0.4166666666666667,
+                                0.3333333333333333,
+                                0.25,
+                                0.0,
+                            ],
+                            y=[
+                                0.35714285714285715,
+                                0.36363636363636365,
+                                0.375,
+                                0.0,
+                            ],
+                            label="b",
+                        ),
+                        Curve(
+                            x=[
+                                0.6923076923076923,
+                                0.6153846153846154,
+                                0.5384615384615384,
+                                0.46153846153846156,
+                                0.38461538461538464,
+                                0.3076923076923077,
+                                0.23076923076923078,
+                                0.07692307692307693,
+                                0.0,
+                            ],
+                            y=[1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0],
+                            label="c",
+                        ),
+                        Curve(x=[1.0, 0.0], y=[1.0, 1.0], label="d"),
+                        Curve(
+                            x=[0.4, 0.3, 0.2, 0.1, 0.0],
+                            y=[0.6666666666666666, 0.6, 0.5, 0.3333333333333333, 0.3333333333333333],
+                            label="e",
+                        ),
+                    ],
+                    x_config=None,
+                    y_config=None,
+                ),
+                ConfusionMatrix(
+                    title="Confusion Matrix",
+                    labels=["a", "b", "c", "d", "e"],
+                    matrix=[[29, 2, 0, 0, 0], [0, 10, 0, 0, 0], [0, 0, 18, 0, 0], [0, 0, 0, 2, 0], [0, 1, 0, 0, 4]],
+                    x_label="Predicted",
+                    y_label="Actual",
+                ),
+            ],
+        ),
+    ],
+)
+def test__prebuilt__object__detection__multiclass__compute__test__case__plots__advanced(
+    config_name: str,
+    expected: Optional[List[Plot]],
+) -> None:
+    config = TEST_CONFIGURATIONS[config_name]
+    eval = ObjectDetectionEvaluator(configurations=[config])
+
+    test_case = TestCase(
+        with_test_prefix(f"complete {config_name} {str(uuid.uuid4())}"),
+        test_samples=[(ts, gt) for _, data in TEST_DATA.items() for ts, gt, _ in data],
+        reset=True,
+    )
+
+    eval.compute_test_sample_metrics(
+        test_case=test_case,
+        inferences=[item for data in TEST_DATA.values() for item in data],
+        configuration=config,
+    )
+
+    result = eval.compute_test_case_plots(
+        test_case=test_case,
+        inferences=[],
+        metrics=[],
```

Comment:
Should these be populated?

Reply:
Might be outdated, but no because the plots rely on caches and don't use those fields.